pipelines/reconfigure.yml

```diff
@@ -333,6 +333,21 @@ jobs:
         concourse_smoke_deployment_name: "concourse-smoke-6-7"
         has_runtime_flag: "true"
         slack_hook: ((slack_hooks.concourse-private))
+        resource_type_versions:
+          bosh-io-release: ~1.0.3
+          bosh-io-stemcell: ~1.0.3
+          cf: ~1.1.0
+          docker-image: ~1.5.1
+          git: ~1.12.1
+          github-release: ~1.5.1
+          hg: ~1.2.0
+          mock: ~0.11.0
+          pool: ~1.1.4
+          registry-image: ~0.14.0
```

Comment:
I'm not sure this will be too useful without setting up the ability to release patch for previous resource-type minors. With the 7.x series we can reasonably say just bump to the latest version, but for 6.7.x there's no way for us to release a patch of the registry-image-resource even if we know it has vulnerabilities

Reply:
Hm yeah that's true - if we ever get to a point where we can support multiple versions of the resource types simultaneously, this'll be more useful. It still does have *some* value now, in that we won't need to manually pin all the resources after releasing